platform-common-testing/src/main/java/com/softicar/platform/common/testing/Asserts.java

```diff
@@ -64,26 +81,52 @@ public static void assertThrows(Class<? extends Throwable> expectedThrowableClas
 					thrownClass.getCanonicalName(),
 					StackTraceFormatting.getStackTraceAsString(thrown)),
 			expectedThrowableClass.isAssignableFrom(thrownClass));
+		if (expectedMessage != null) {
+			assertEquals("Unexpected message.", expectedMessage, thrown.getMessage());
```

Comment:
Maybe `thrown.getLocalizedMessage()` because subclasses of `Throwable` usually overwrite that method(?)